tools/lib/clean_duplicate_images.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
+#
+# Copyright 2016 Canonical Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+A tool to delete duplicate glance images not in use.
```

Comment:
This docstring does not folllow python standards - """ and "A Tool" should be on one line.